### DISCUSSION TITLE
Timeout expired. IDE unable to establish connection.

### DISCUSSION THREAD

**Discussion by eSm1ley** [apache/netbeans#7406] [General]
### Apache NetBeans version

Apache NetBeans 21

### What happened

I was trying to add a new connection specifically a JDBC connection but it keeps on having an error about "Timeout expired. IDE unable to establish connection. Check your proxy settings or try again later. The server may be unavailable at the moment.   You may also want to make sure that your firewall is not blocking network traffic". I've done multiple of ways to fix it and follow guidelines on how to make it work but it's not working 

### Language / Project Type / NetBeans Component

Java Application Project and JDBC Connection

### How to reproduce

I went to Services.
Right clicked the database to create a new connection
Added a new driver named "mssql-jdbc-12.6.2.jre11.jar"
After that I fill up the necessary textfields
Then right after a few minutes "Timeout expired. IDE unable to establish connection. Check your proxy settings or try again later. The server may be unavailable at the moment.   You may also want to make sure that your firewall is not blocking network traffic."
I made lots of ways to fix it but it keeps on saying "Timeout expired. IDE unable to establish connection. Check your proxy settings or try again later. The server may be unavailable at the moment.   You may also want to make sure that your firewall is not blocking network traffic."

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Window 11 23H2

### JDK

JKD 21

### Apache NetBeans packaging

Apache NetBeans binary zip

### Are you willing to submit a pull request?

No

**Comment by matthiasblaesing** [comment]
The message is pretty clear: You can't connect to the DB. I worked with MSSQL in the past and I had to adjust the DB server to give me stable access. More info might be found in the IDE log (in Menu: View -> IDE Log).
